php/Dockerfile

```diff
@@ -5,4 +5,6 @@ RUN echo $(php --version)
 ADD . /app
 WORKDIR /app
 
+RUN sed -i 's/CipherString = DEFAULT@SECLEVEL=2/CipherString = DEFAULT@SECLEVEL=1/g' /etc/ssl/openssl.cnf
```

Comment:
with `CipherString = DEFAULT@SECLEVEL=2` out python, ruby and php example fails because `key is weak`. we need to  upgrade our certificate